mlbparks-template-eap copy-4.4.json

```diff
@@ -0,0 +1,463 @@
+{
```

Comment:
sure the file should be named `*.copy-4.4.json` ?  `s/copy//g` ?

Reply:
Fixed, so much thank you for review!!